projects/ngx-interactive-paycard-demo/src/assets/credit-card.ts

```diff
@@ -0,0 +1,71 @@
+export const creditCards: any = [
+  {
+    name: 'American Express',
+    startsWith: ['34', '37'],
+    length: [15],
```

Comment:
There is a general problem with the card number length and card number format. For example, AMEX cards have `####-######-#####` format so their length is 15. On the card the `####-####-####-####` format is fixed with 16 length, so it looks like this when a complete AMEX card number is added:
![image](https://user-images.githubusercontent.com/4102195/101277691-7adfe080-37b6-11eb-82ed-b72f2d1e84d7.png)

This is not good this way, because the card format is confusing. In this case the card number format should be also dynamic, is should align to the currently used card format.

An idea to solve this is, to store in this `creditCards` the `cardNumberFormat` and `cardNumberMask` as well, and add them as input for library. (The input properties `cardNumberFormat` and `cardNumberMask` already exist.)